src/Rest/RestApiBrowser.php

```diff
@@ -82,7 +76,7 @@ public function getRequestHeaders()
     }
 
     /**
-     * @return HttpClient
+     * @return Http\Client\HttpClient
      */
     public function getHttpClient()
```

Comment:
Do we still need it? Not used anywhere in the project and we have no reason to expose it once we have some or specified we use some another http client.